backend/src/v4/models/bcf.js

```diff
@@ -715,7 +715,8 @@ async function parseViewpointComponents(groupDbCol, vpComponents, isFederation,
 									parseViewpointComponentIfc(groupDbCol.model, component, ifcToModelMap, isFederation)
 								);
 							}
-							if (vpComponents[componentsIdx][componentType][i].Exceptions) {
+							if (vpComponents[componentsIdx][componentType][i].Exceptions &&
```

Comment:
you should be able to just do this

```suggestion
							if (vpComponents[componentsIdx][componentType][i].Exceptions?.Component
```